src/Tizen.NUI.Components/Controls/Menu.cs

```diff
@@ -121,6 +122,7 @@ public IEnumerable<MenuItem> Items
         /// If there is no enough space to display menu at the anchor position,
         /// then menu is displayed at the proper position near anchor position.
         /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Should developers set position manually?  
If the required functionality is similar to Android "PopupMenu", anchor view concept is more suitable to calculate position of the menu automatically.

Reply:
I think that there are 2 ways of usages.
- Showing menu at a given object (such as drop down button)
  - "Anchor" property may be useful.
- Showing menu at a given point (such as copy paste menu)
  - "Anchor Position" property may be useful.

I think that the properties "Anchor" and "Anchor Position" affect each other.
(For Android case, they use only "Anchor")

I think that "Anchor Position" may cover "Anchor" as well because we can set the proper position of "Anchor" to "Anchor Position". (but I think that "Anchor" cannot cover "Anchor Position")
However, it is not so easy to get and set "Anchor"'s position to "Anchor Position".

So I adds the static methods for menu as follows in DialogPage.
- public static void ShowMenu(View anchor, params MenuItem[] items)
- public static void ShowMenu(Position2D anchorPosition, params MenuItem[] items)
```
DialogPage.ShowMenu(anchor, menuItem, menuItem2, ...);
```

If you think that "Anchor" property is better than "Anchor Position", then I can provide "Anchor" property instead of "Anchor Position".
But I am not sure how to support "Anchor Position" usage with "Anchor" property. I need to think about it more.

Reply:
I have added "Anchor" property and removed "AnchorPosition" property.